src/app/core/components/header/header.component.scss

```diff
@@ -17,13 +13,17 @@
   flex-shrink: 0;
 }
 
-.browse-category {
-  border-right: 1px solid var(--border-color);
-  white-space: nowrap;
+.header-content {
   display: flex;
   align-items: center;
```

Comment:
Use 'column-gap' (or gap) instead of margin on child element to define additional space between elements.